src/Mono.Android/Xamarin.Android.Net/AndroidMessageHandler.cs

```diff
@@ -345,7 +383,10 @@ string EncodeUrl (Uri url)
 			}
 		}
 
-		internal virtual async Task <Java.Net.Proxy?> GetJavaProxy (Uri destination, CancellationToken cancellationToken)
+		internal Task <HttpResponseMessage> SendAsyncInternal (HttpRequestMessage request, CancellationToken cancellationToken)
+			=> SendAsync (request, cancellationToken);
+
+		protected virtual async Task <Java.Net.Proxy?> GetJavaProxy (Uri destination, CancellationToken cancellationToken)
```

Comment:
I don't understand changes "like this", in which an `internal virtual` method is turned into a `protected virtual` + `internal` method pair.  This increase "supported API surface area", without providing docs for the newly added members.

Is this for compatibility with Legacy `AndroidClientHandler`, which *does* have a `protected virtual GetJavaProxy` method?

A quick spot-check suggests that these changes *do* increase compatibility with Legacy `AndroidClientHandler`, which just "redirects" the question: why were these methods changed *from* `protected virtual` to `internal virtual` in 1e5bfa3ff0ae18ce9407677667f05a9bfba53478?  @steveisok?  Is there something we should be aware of here?

Reply:
https://discord.com/channels/732297728826277939/732297837953679412/920360699711090768

@steveisok wrote:

> I believe it's as simple as we didn't consider subclassing of `AndroidMessageHandler` at the time.  I believe "undoing" is to make sure we have the least amount of friction.

Reply:
Beat me to it ;-)

Reply:
> without providing docs for the newly added members

Just a comment regarding the docs: I fixed all comments in `AndroidMessageHandler` (they were all copied from `AndroidClientHandler` and they often still contained references to `AndroidClientHandler`) so the docs of the protected functions will be correct. The `GetJavaProxy` fn doesn't have any docs in [AndroidClientHandler](https://docs.microsoft.com/en-us/dotnet/api/xamarin.android.net.androidclienthandler?view=xamarin-android-sdk-12) so I didn't try to come up with any new documentation for the function.